lib/activation-service.js

```diff
@@ -11,13 +11,15 @@ const activationService = {};
 
 activationService.run = () => {
     activationService.pollActivation()
-        .then(() => {
+        .then((response) => {
             Store.setDeviceStatus(DeviceStatus.ACTIVE);
-            logger.success('Activation successful. Triggering actions enabled');
+            logger.success('Activation successful for ' + response.message
+                                            +'. Triggering actions enabled');
```

Comment:
something went wrong with the indentation I think. Not a dealbreaker

Reply:
Because there is addition of response.message. Since the line length crossed 80, I moved the extra to next line